Src/Extern/amrdata/AMReX_DataServices.cpp

```diff
@@ -2488,7 +2488,7 @@ void DataServices::RunTimelinePF(std::map<int, std::string> &mpiFuncNames,
           int ntnSize(nameTagNames.size());
           ntnNumbers.resize(ntnSize, 0.0);
           if(ntnSize > 0) {
-            ntnMultiplier = pow(10, static_cast<int>( 1 + log10(ntnSize)));
+            ntnMultiplier = std::pow(10, static_cast<int>( 1 + log10(ntnSize)));
```

Comment:
Let's also update log10 here and the change below.

Reply:
Weird that this didn't cause a compile error as for `pow()`.